src/plugin/ui/theme.test.ts

```
@@ -0,0 +1,88 @@
+      expect(theme.glyphs.selected).toBe('>');
+      expect(theme.colors.reset).toBe('\x1b[0m');
+    });
+
+    it('produces different color codes for each profile', () => {
+      const truecolor = createUiTheme({ profile: 'truecolor' });
+      const ansi256 = createUiTheme({ profile: 'ansi256' });
+      const ansi16 = createUiTheme({ profile: 'ansi16' });
+
+      expect(truecolor.colors.primary).toContain(';2;');
+      expect(ansi256.colors.primary).toContain(';5;');
+      expect(ansi16.colors.primary).toMatch(/^\x1b\[\d+m$/);
```

Biome (2.4.4)

[error] Unexpected control character in a regular expression.

(lint/suspicious/noControlCharactersInRegex)